### PR TITLE
fix left click attack when right click is hidden

### PR DESCRIPTION
**Pull request by Kyleeld**
fix left click attack when right click is hidden

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/clanmanmode/ClanManModePlugin.java [hunk 1]
```diff
@@ -153,7 +164,18 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 					}
 				}
 			}
-		}
+		}	
+
+	}
+
+	void startPrioritizing()
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/clanmanmode/ClanManModePlugin.java]: What is the purpose of this method?

File: runelite-client/src/main/java/net/runelite/client/plugins/clanmanmode/ClanManModePlugin.java [hunk 2]
```diff
@@ -153,7 +164,18 @@ public void onMenuEntryAdded(MenuEntryAdded event)
...
+	{
+		menuManager.addPriorityEntry(WALK_HERE);
+	}
+
+	void stopPrioritizing()
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/clanmanmode/ClanManModePlugin.java]: And this one?

**Comment by Kyleeld** [comment]
fixed @Ganom

**Comment by Ganom** [comment]
Have you tested this ingame?

**Comment by Kyleeld** [comment]
only if hide attack is on, which is fine

**Comment by Ganom** [comment]
So you want to completely hide every single menu in the game if that config is enabled?

**Pull request closed without merging**